src/Umbraco.Web.BackOffice/Trees/ContentTreeControllerBase.cs

```diff
@@ -159,7 +159,21 @@ public abstract class ContentTreeControllerBase : TreeController, ITreeNodeContr
     {
         var entityIsAncestorOfStartNodes =
             ContentPermissions.IsInBranchOfStartNode(e.Path, startNodeIds, startNodePaths, out var hasPathAccess);
-        var ignoreUserStartNodes = IgnoreUserStartNodes(queryStrings);
+
+        bool ignoreUserStartNodes;
```

Comment:
I don't think is a good option to ignore the start nodes, this will allow users to see the content that they were not able to see before and restore it without the possibility to access it